napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -311,14 +498,22 @@ def addItem(
         if project_info.home_page:
             import webbrowser
 
-            widg.help_button.clicked.connect(
+            widg.plugin_name.clicked.connect(
                 partial(webbrowser.open, project_info.home_page)
             )
```

Comment:
Partial may lead to leak memory when connecting to Qt signals. Please add at least FIXME comment.

Reply:
These partials were added as part of another PR.  I will add a FIXME comment for now.